.travis.yml

```diff
@@ -2,23 +2,53 @@ language: php
 
 sudo: false
 
-php:
-  - 5.5
-  - 5.6
-  - 7.0
-  - hhvm
+cache:
+    directories:
+        - $HOME/.composer/cache/files
+env:
+    global:
+        - TEST_COMMAND="composer test"
 
 matrix:
-  include:
-    - php: 5.5
-      env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
+    fast_finish: true
+    include:
+          # Minimum supported PHP and Symfony version
+        - php: 5.5
+          env: DEPENDENCIES="minimun"
 
-before_script:
-  - git config --global user.name travis-ci
-  - git config --global user.email travis@example.com
+          # Test the latest stable release
+        - php: 5.5
+        - php: 5.6
+        - php: 7.0
+        - php: 7.1
+        - php: hhvm
+
+          # Test LTS versions
+        - php: 7.1
```

Comment:
imo we don't need to increase number of jobs.
5.5 - minimum
5.6 - v2
7.0 - v3
7.1 - beta

Reply:
I agree with you. I was not sure you wanted to remove so many PHP versions